new-frontiers-14/frontier-station-14
# Show docking labels, order labelled docks

## About the PR



Returns docking labels to the shuttle console's nav screen, the radar console, and the STC's console displays docks in alphabetical order.

Left them white - tried a set of light blue colours but wasn't thrilled with how they turned out.

Trade Outpost's docks have been labelled - CB1-4 to match the signs, smaller docks labelled D5-7 in clockwise order continuing from 4 (leaving D7 to be the bus stop)

## Why / Balance


~~Fumbled~~ Lost technology, STC gaming.

## How to test


Check aghost radar, check shuttle consoles, check the STC's remote and local channels - both the nav and the docking screens.

## Media


Shuttle console, showing docking labels at Frontier Outpost:
![image](https://github.com/user-attachments/assets/bb89eef7-dfbd-47c4-8d74-159d109763a6)

STC's console around Frontier Outpost
![image](https://github.com/user-attachments/assets/5aa09261-705a-408c-9f82-fe5c882b7ada)

STC's remote console for the Trade Outpost.  Cargo bays are labelled as "CB#", docks as "D#" (considered D7B to denote where the bus normally spawns, decided against it) - edited after swapping erroneous CB3/D5 labels
![image](https://github.com/user-attachments/assets/ca9c441a-6ed2-4e49-a38d-739588a9ffdd)

STC's remote console showing the dock list.  All docks are labelled, and listed in alphabetical order.
![image](https://github.com/user-attachments/assets/8fd262a1-54de-4f90-8aa7-47f270a8859b)

## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Returned dock labels, labelled Trade Outpost's docks.
